web/blueprints/finance/forms.py

```diff
@@ -7,6 +7,7 @@
 from wtforms import Form as WTForm, ValidationError
 from wtforms.validators import DataRequired, NumberRange, Optional, \
     InputRequired
+from wtforms.widgets import TextArea
```

Comment:
Why not use our decorated TextAreaField?
https://github.com/agdsn/pycroft/blob/ac8be4c9c49ac6ae5a005a08b152625e65921b39/web/form/fields/core.py#L192-L198